pkg/apis/garden/validation/validation.go

```diff
@@ -1426,10 +1426,37 @@ func ValidateShootSpecUpdate(newSpec, oldSpec *garden.ShootSpec, deletionTimesta
 
 	allErrs = append(allErrs, validateDNSUpdate(newSpec.DNS, oldSpec.DNS, fldPath.Child("dns"))...)
 	allErrs = append(allErrs, validateKubernetesVersionUpdate(newSpec.Kubernetes.Version, oldSpec.Kubernetes.Version, fldPath.Child("kubernetes", "version"))...)
+	allErrs = append(allErrs, validateKubeProxyModeUpdate(newSpec.Kubernetes.KubeProxy, oldSpec.Kubernetes.KubeProxy, newSpec.Kubernetes.Version, fldPath.Child("kubernetes", "kubeProxy"))...)
 
 	return allErrs
 }
 
+func validateKubeProxyModeUpdate(new, old *garden.KubeProxyConfig, version string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	newMode := garden.ProxyModeIPTables
+	oldMode := garden.ProxyModeIPTables
+	if new != nil {
```

Comment:
Why is this duplicated? This stuff is already validated [here](https://github.com/gardener/gardener/blob/master/pkg/apis/garden/validation/validation.go#L1717) and is executed also for update requests.